src/pages/gyeongminLee/Login/Login.js

```diff
@@ -1,8 +1,151 @@
+      }),
+    })
+      .then(response => response.json())
+      .then(
+        result => {
+          if (result.MESSAGE !== 'SUCCESS') {
+            //응답은 맞는경우도 있고 아닌경우도 있다 - if문으로 나누어 주어야된다
+            this.props.history.push('/main-Gyeongmin');
+            //alert(result.ACCESS_TOKEN);
+            localStorage.setItem('Token', result['ACCESS_TOKEN']);
+          }
+        }
+
+        //.then setItem('ACCESS_TOKEN', result.ACCESS_TOKEN)
+      );
+    // .then(localStorage.setItem('ACCESS_TOKEN',  JsonResponse.ACCESS_TOKEN));
+  };
+
   render() {
-    return <div>로그인</div>;
+    //console.log(JsonResponse['ACCESS_TOKEN']);
+    // const idValid = this.state.idValue.length >= 1; 멘토님의 것
+    //const inputValid =this.state.idValue.includes('@')&&this.state.password.length>=5
+    const idValid = this.state.idValue.includes('@');
+    const passwordValid = this.state.passwordValue.length >= 5;
+    // let button = null;
+
+    // if (idValid && passwordValid) {
+    //   button = (
+    //     <button
+    //       //onClick={this.goToMain}
+    //       type="button"
+    //       id="loginButton"
+    //       backgroundColor="#0095F6"
+    //     >
+    //       {' '}
+    //       로그인
+    //     </button>
+    //   );
+    // } else {
+    //   button = (
+    //     <button
+    //       //onClick={this.goToMain}
+    //       type="button"
+    //       id="loginButton"
+    //       backgroundColor="#B2DFFC"
+    //     >
+    //       {' '}
+    //       로그인
+    //     </button>
+    //   );
+    // }
+
+    return (
+      <>
+        <div className="container">
+          <form id="logo" action="index.html" method="post">
+            <a id="logo">westagram</a>
+          </form>
+
+          <form id="login-form" onChange={this.controlValue}>
+            <input
+              type="text"
+              className="text"
+              id="id"
+              placeholder="전화번호,사용자 이름 또는 이메일"
+            />
+            <input
+              type="password"
+              className="text"
+              id="password"
+              placeholder="비밀번호"
+            />
+
+            <button
+              //style={{ backgroundColor: this.state.buttonColor }} // 클래스 네임을 동적으로 바꾸어주자(무슨뜻인지 모름 ^^)
+              //disabled
+              //
+
+              //disabled={true}
+              //button={button}
+              // onChange={function () {
+              //   if (idValid && passwordValid) {
+              //     button.backgroundColor = '#0095F6';
+              //   }
+              // }}
+              // style={{
+              //   backgroundColor: '#B2DFFC',
+              // }}
+              onClick={this.goToMain}
+              type="button"
+              id="loginButton"
+              backgroundColor={
+                idValid && passwordValid === true ? '#B2DFFC' : '#0095F6'
+              }
+              disabled={idValid && passwordValid === true ? false : true}
+            >
```

Comment:
삼항연산자 안쓰고 boolean의 특성을 이용해서 간결하게 표현 가능해보입니다!
```suggestion
              backgroundColor={
                idValid && passwordValid ? '#B2DFFC' : '#0095F6'
              }
              disabled={!(idValid && passwordValid)}
            >
```